src/core.js

```diff
@@ -86,8 +86,6 @@ extend( QUnit, {
 
 	only: only,
 
-	// DEPRECATED: The functionality of QUnit.start() will be altered in QUnit 2.0.
-	// In QUnit 2.0, invoking it will ONLY affect the `QUnit.config.autostart` blocking behavior.
```

Comment:
not yet, something that wasn't so simple to remove. I need to rework the "promiseable tests"  api. I'm working to get it done.